lib/rebay_api_web/controllers/session_controller.ex

```diff
@@ -26,7 +28,7 @@ defmodule RebayApiWeb.SessionController do
   def delete(conn, _params) do
     conn
     |> configure_session(drop: true)
-    |> redirect(to: Routes.item_path(conn, :index))
+    |> redirect(external: "#{@client_host}")
```

Comment:
I'd guess it's probably more idiomatic to route to a Phoenix path, rather than your services configured host and port.